lib/member_records.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+require 'scraped'
+require_relative 'member_record'
+
+class MemberRecords < Scraped::JSON
+  field :members do
+    all_members
```

Comment:
it's not really clear what this field is, or why, if it _is_ needed, it does nothing but call a private method.